src/pages/CreatorCenter/components/Main/components/MainContent/components/ClassListSearchBar/ClassListSearchBar.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function ClassListSearchBar(props) {
+  return (
+    <SearchBar>
+      <SearchInput />
```

Comment:
```suggestion
      <SearchInput placeholder="상품명 검색" />
```
이렇게 작성하면 아래 스타일에 attrs를 제거해도 되겠습니다!